mockserver.js

```diff
@@ -60,11 +61,73 @@ var parse = function (content) {
         }
     });
 
-    body = bodyContent.join('\n');
+    body = applyHooks(bodyContent).join('\n');
```

Comment:
Instead of applying the hooks to the list of lines, why not process the content as a template?

ie.

```
body = swig.render(bodyContent.join('\n'))
```

and you define the time function as a swig function. I think templating is a better fit for this problem so that we dont have our own logic to parse the line, check if it's a function, extract its arguments etc etc